packages/block-library/src/image/view.js

```diff
@@ -83,36 +83,54 @@ const { state, actions, callbacks } = store(
 			},
 		},
 		actions: {
-			showLightbox() {
+			hideLightbox( event ) {
+				/*
+				 * This handler is purely for the custom action of hiding the lightbox when clicking anywhere inside.
+				 * If this handler is triggered due to a click on the close button, it should be ignored as that
+				 * button automatically hides the lightbox via `popovertarget` and `popovertargetaction`.
+				 */
+				if ( event?.target?.tagName === 'BUTTON' ) {
```

Comment:
isn't the entire image a button now? does this bit still work/is it still needed?